src/client/swagger/managedFunctions.yml

```diff
@@ -167,57 +167,94 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/api/v2private/functions/{functionID}/runs':
+  '/api/v2/functions/{functionID}/invoke':
     get:
-      operationId: GetFunctionsIDRuns
+      operationId: GetFunctionsIDInvoke
       tags:
         - Functions
-      summary: List runs for a function
+      summary: Manually invoke a function with params in query
       parameters:
         - in: path
           name: functionID
           schema:
             type: string
           required: true
-          description: The ID of the function to get runs for.
+        - in: query
+          name: params
+          schema:
+            type: object
+            additionalProperties: true
+          style: form
+          explode: true
       responses:
         '200':
-          description: A list of function runs
+          description: Function invocation response
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/FunctionRunRecords'
+                $ref: '#/components/schemas/FunctionHTTPResponseData'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
     post:
-      operationId: PostFunctionsIDRuns
+      operationId: PostFunctionsIDInvoke
       tags:
         - Functions
-      summary: Manually trigger a function run
+      summary: Manually invoke a function with params in request body
       parameters:
         - in: path
           name: functionID
           schema:
             type: string
           required: true
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/FunctionInvocationParams'
       responses:
         '201':
-          description: Run triggered
+          description: Function invocation response
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/FunctionRun'
+                $ref: '#/components/schemas/FunctionHTTPResponseData'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  '/api/v2/functions/{functionID}/runs':
```

Comment:
Did we remove POST to `/api/v2/functions/{functionID}/runs` to trigger a new run of a stored function?

I suppose we don't need it for the UI - as the UI always calls the `trigger` URL, but I am wondering if this is intended.

Reply:
yes, I replaced POST to `/api/v2/functions/{functionID}/runs` with GET or POST to `/api/v2/functions/{functionID}/invoke` for now.